strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.xml

```diff
@@ -49,8 +49,7 @@
                     <remote-repository url="https://repo.maven.apache.org/maven2/"
                                        download-remote-indexes="true"
                                        auto-blocking="true"
-                                       checksum-validation="true"
-                                       check-interval-seconds="20" />
```

Comment:
@carlspring I think that we should use default 60 seconds interval here too. It was added for test purposes.

Reply:
Okay, I agree.